genny/auth/auth.go

```diff
@@ -0,0 +1,152 @@
+package auth
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/gobuffalo/buffalo-auth/transformer"
+	"github.com/gobuffalo/buffalo/generators"
+	"github.com/gobuffalo/buffalo/meta"
+	"github.com/gobuffalo/genny"
+	"github.com/gobuffalo/genny/movinglater/plushgen"
+	"github.com/gobuffalo/packr"
+	"github.com/gobuffalo/plush"
+	"github.com/markbates/inflect"
+	"github.com/pkg/errors"
+)
+
+// New actions/auth.go file configured to the specified providers.
+func New(args []string) (*genny.Generator, error) {
+	g := genny.New()
+
+	fields, extraFields := extractFields(args)
```

Comment:
Can you use https://godoc.org/github.com/gobuffalo/genny/movinglater/attrs here?